lib/node/node.js

```diff
@@ -471,9 +478,8 @@ function validate(validator, params, expectedParamName) {
 }
 
 Node.prototype.checkLimits = function(context, callback) {
-    // the default is to limit number of output rows if a maximumNumberOfRows is
-    // present in the configuration (not by default)
-    var defaultLimit = null;
-    var limit = context.getLimit(this.getType(), 'maximumNumberOfRows', defaultLimit, 'too many result rows');
-    limits.limitNumberOfRows(this, context, limit, callback);
+    Checks.check(this, context, [{
```

Comment:
Do we want to have that default limit? Or should it be implicit?

I mean: could we just `return callback(null);`.